group_theory/perm.lean

```diff
@@ -261,12 +330,42 @@ def sign [fintype α] (f : perm α) := sign_aux3 f mem_univ
 instance sign.is_group_hom [fintype α] : is_group_hom (@sign α _ _) :=
 ⟨λ f g, (sign_aux3_mul_and_swap f g _ mem_univ).1⟩
 
+@[simp] lemma sign_mul [fintype α] (f g : perm α) : sign (f * g) = sign f * sign g :=
+is_group_hom.mul sign _ _
+
+@[simp] lemma sign_one [fintype α] : (sign (1 : perm α)) = 1 :=
+is_group_hom.one sign
+
+@[simp] lemma sign_inv [fintype α] (f : perm α) : sign f⁻¹ = sign f :=
+by rw [is_group_hom.inv sign, int.units_inv_eq_self]; apply_instance
+
 lemma sign_swap [fintype α] {x y : α} (h : x ≠ y) : sign (swap x y) = -1 :=
 (sign_aux3_mul_and_swap 1 1 _ mem_univ).2 x y h
 
 lemma sign_eq_of_is_swap [fintype α] {f : perm α} (h : is_swap f) : sign f = -1 :=
 let ⟨x, y, hxy⟩ := h in hxy.2.symm ▸ sign_swap hxy.1
 
+lemma sign_symm_trans_trans [fintype α] [decidable_eq β] [fintype β] (f : perm α)
+  (e : α ≃ β) : sign ((e.symm.trans f).trans e) = sign f :=
+show sign_aux3 ((e.symm.trans f).trans e) (show ∀ x : β, x ∈ univ.1, from mem_univ) =
```

Comment:
would it make sense to have a separate lemma about `sign_aux3`. Where the `mem_univ` proofs are replaced by assumptions?